docs/source/tutorials/videos/how-to.md

```diff
@@ -0,0 +1,42 @@
+# "How to?" Tutorial Series
+
+This tutorial series is meant to give you a hands-on experience of the Nipoppy toolbox. Please have Nipoppy installed on your device (see [Installation](../../overview/installation.md)) and have our example dataset ready:
+
+```{code-block} console
+$ git clone https://github.com/nipoppy/tutorial-dataset.git
+```
+
+or visit the [GitHub repo](https://github.com/nipoppy/tutorial-dataset) and download the data without using `git`. We show in the videos how to!
+
+## 1. nipoppy init
+
+In this tutorial we will cover how to create a new Nipoppy dataset. More concretely, we will
+- run the `nipoppy init` command
+- discover the directories that follow the Nipoppy specification
+- explore the `nipoppy status` command
+- modify the content of the `manifest.tsv` file according to our dataset
+- and run the `nipoppy track-curation --regenerate` command
```

Comment:
Would be great to have links to the cli doc for each command and concept

e.g., 
- `nipoppy init` -> https://nipoppy--740.org.readthedocs.build/en/740/cli_reference/init.html
- `manifest.tsv` -> https://nipoppy--740.org.readthedocs.build/en/740/explanations/manifest.html